test/e2e/dra/README.md

```diff
@@ -21,3 +21,6 @@ built from their main branch, use:
 
     test/e2e/dra/kind-build-image.sh dra/node:latest && \
     kind create cluster --config test/e2e/dra/kind.yaml --image dra/node:latest
+
+
+NB: For not amd64 cpu architecture you might want to have a look [here](https://github.com/kind-ci/containerd-nightlies/releases/tag/containerd-1.6.0-830-g34d078e99) and update these [lines](https://github.com/kubernetes/kubernetes/blob/master/test/e2e/dra/kind-build-image.sh#L35-L36).
```

Comment:
Would it make sense to change `test/e2e/dra/kind-build-image.sh` to use current arch?

Reply:
It was my first intent but `uname -i` on my mac returns `unknown`. I can dig more to find a way to use the current architecture as long as it matches `amd64`, `arm64`, `ppc64le` or `s390x`. I don't know if there are others ... but the release `containerd-1.6.0-830-g34d078e99` only supports these ones.

Reply:
try `go env GOARCH`

Reply:
You can use newer version of containerd. BTW, the latest one has builds only for 2 archs: https://github.com/kind-ci/containerd-nightlies/releases/tag/containerd-1.7.0-79-g2503bef58

Reply:
will do ! yes `go env GOARCH` should do the trick